api/app/services/searcher.py

```diff
@@ -2,36 +2,36 @@
 
 from pyserini.search import pysearch
 
-from app.models import SearchVertical
 from app.settings import settings
 
 
 class Searcher:
     def __init__(self):
-        self.searchers: List[pysearch.SimpleSearcher] = {}
-        self.searchers[SearchVertical.cord19] = self.build_searcher(
-            settings.cord19_index_path)
-        #self.searchers[SearchVertical.trialstreamer] = self.build_searcher(
-        #    settings.trialstreamer_index_path)
+        self.build_searcher(settings.index_path)
```

Comment:
if `self.searchers` is no longer a list, does it mean that covidex only supports searching 1 corpus atm?

Reply:
Yeah, easy to change later but this is cleaner

Reply:
I don't think any deployment would need multiple corpora at the same time